subxt/src/config/polkadot.rs

```diff
@@ -11,6 +11,7 @@ use crate::SubstrateConfig;
 pub use primitive_types::{H256, U256};
 
 /// Default set of commonly used types by Polkadot nodes.
+#[derive(Debug)]
```

Comment:
We shouldn't need this; this is because deriving Debug on `Foo<T: Config>` will automatically expect it on T, too. We work around this by either using Derivative to say that we don't care about `T`, or manually implementing something to ignore T :)

Reply:
(It's more annoying to do it this way, but more "correct" as well :))